pyteal/ast/itxn.py

```diff
@@ -200,6 +206,111 @@ def SetFields(cls, fields: Dict[TxnField, Union[Expr, List[Expr]]]) -> Expr:
         fieldsToSet = [cls.SetField(field, value) for field, value in fields.items()]
         return Seq(fieldsToSet)
 
+    @classmethod
+    def MethodCall(
```

Comment:
Worthwhile having some kind of unit test or even a graviton test that shows how to use it. It could be based on: 
<img width="649" alt="image" src="https://user-images.githubusercontent.com/291133/172730564-9c3d7759-05da-4f03-b74c-a7568d76de1d.png">

I suspect that graviton will fall flat because of inner txn's, but would love to be proved wrong or at least have a good example to aim for when inner txn's are supported.